packages/documentation/src/stories/utilities/sizing/sizing.stories.ts

```diff
@@ -1,15 +1,101 @@
-import { Meta, StoryObj } from '@storybook/web-components';
+import type { Args, Meta, StoryObj, StoryContext } from '@storybook/web-components';
+import { html } from 'lit';
 import { BADGE } from '../../../../.storybook/constants';
+import './sizing.styles.scss';
+import scss from './sizing.module.scss';
+
+export const SCSS_VARIABLES = scss;
+
+const sizingOptions = ['auto', ...Object.keys(SCSS_VARIABLES)];
+
+const sizeOptionsPercent = ['auto', '25', '50', '75', '100'];
 
 const meta: Meta = {
+  render: renderSizing,
   title: 'Utilities/Sizing',
   parameters: {
-    badges: [BADGE.TODO],
+    badges: [BADGE.NEEDS_REVISION],
+  },
+  args: {
+    height: 'bigger-giant',
+    width: 'bigger-giant',
+    border: false,
+  },
+  argTypes: {
+    border: {
+      control: {
+        type: 'boolean',
+      },
+      table: {
+        disable: true,
+      },
+    },
+    height: {
+      name: 'height',
+      description: 'Set the height of the cube',
+      control: {
+        type: 'select',
+      },
+      options: sizingOptions,
+    },
+    width: {
+      name: 'width',
+      description: 'Set the width of the cube',
+      control: {
+        type: 'select',
+      },
+      options: sizingOptions,
+    },
+    maxHeight: {
+      name: 'max-height',
+      description: 'Set the maximum height of the cube',
+      control: {
+        type: 'select',
+      },
+      options: sizingOptions,
+    },
+    maxWidth: {
+      name: 'max-width',
+      description: 'Set the maximum width of the cube',
+      control: {
+        type: 'select',
+      },
+      options: sizingOptions,
+    },
   },
 };
 
 export default meta;
 
 type Story = StoryObj;
 
-export const Default: Story = {};
+function renderSizing(args: Args) {
+  const classes = `content h-${args.height} w-${args.width} mh-${args.maxHeight} mw-${args.maxWidth}`;
```

Comment:
`args.maxHeight` and `args.maxWidth` can be undefined, resulting in adding class like `mh-undefined`